wgsl/index.bs

```diff
@@ -5269,6 +5283,30 @@ Example: `vec3(x,x,x)` is analyzed as follows:
 * The type of the expression is a [=vector=] of 3 components
     of [=i32=] (`vec3<i32>`).
 
+<div class='example override-expression errors' heading='Pipeline-creation errors from override-expressions'>
+  <xmp highlight=wgsl>
+  override a : i32 = 0;
+  override b = 1 / a;
+  override c = a / 0; // shader-creation error, regardless of attempting to override c
```

Comment:
It may be a minor detail but if c is always a shader-creation error, it makes reading other comments a bit more confusing as it might feel like the code is made to fail but there are cases. Could this be an additional example? Maybe just `a` and `c`.

Reply:
I had thought about that too. I agree you sort of have to read it in pieces currently. My thought was it might be more meaningful included together, but you'd never reach a pipeline compile. Maybe splitting it up is best.